assets/sass/components/settings/_googlesitekit-settings-notice.scss

```diff
@@ -53,6 +52,7 @@
 	.googlesitekit-settings-notice__text {
 		flex-grow: 1;
 		line-height: 20px; // Match icon height.
+		max-width: 1098px;
```

Comment:
Let's add a new variable for this so that it isn't such an arbitrary number.

Reply:
Added, thanks!